Flash Chat iOS13/Controllers/ThreadListViewController.swift

```diff
@@ -73,6 +104,8 @@ class ThreadListViewController: UITableViewController {
 						}
 					}
 				}
+			}
+					}
```

Comment:
⚒️ 
```suggestion
			}
		}
```

Reply:
I have since removed that array as it didn't store anything. I was intending to append to it as it fetched from the server, but since I'm only ever fetching one user at a time I didn't need to do that.

Reply:
Hmm, you're replying to my change request that was just about a bunch of extra tabs. Did you mean to place this comment elsewhere?

Reply:
I don’t understand. Where else would I have put it?

Reply:
It only fetches users whose email addresses match the logged in user (there should only be one). It isn't fetching all of them.
Sorry if I'm putting comments in the wrong places, don't know how this all works...